public/js/shopping-list-item.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    this.isDone = true;
+  }
+
+  unCheck() {
+    this.isDone = false;
+  }
+
+  render(i) {
+    var test= `<li class = "completed_${this.isDone}"> <input type='checkbox' onchange= 'changeCheckedStatus(${i}, this)'/><span>${
```

Comment:
Use const or let instead of var. Also, try to handle whether the checkbox is checked here in the string: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value